projects/kit/components/input-tag/input-tag.style.less

```diff
@@ -66,9 +71,14 @@
     padding: 0;
     align-items: flex-start;
 
+    :host[data-tui-host-size='s'] &,
     :host[data-tui-host-size='m'] & {
         padding-left: 0.75rem;
     }
+
+    :host[data-tui-host-size='s'] & {
+        padding-right: 0;
```

Comment:
Do we need it? It looks like it's 0 anyway

Reply:
yes, we need it because of [this line](https://github.com/TinkoffCreditSystems/taiga-ui/blob/main/projects/core/styles/mixins/textfield.less#L37)